.github/workflows/docker-build-test-upload.yml

```diff
@@ -11,10 +11,18 @@ on:
         description: Parent image name
         required: true
         type: string
+      parent-variant:
+        description: Parent variant tag prefix
+        required: true
+        type: string
```

Comment:
I suggest making `default` the default value here 😃.
Currently, parent image is always default, so we will add the option if someone needs it in the future, but `docker-build-test-upload-all.yml` will be slightly more simple.

Reply:
Added `default` as the default value.